capz/run-capz-e2e.sh

```diff
@@ -36,10 +37,12 @@ main() {
     if [[ "${GMSA}" == "true" ]]; then create_gmsa_domain; fi
 
     create_cluster
-    apply_workload_configuraiton
-    wait_for_nodes
-    if [[ "${HYPERV}" == "true" ]]; then apply_hyperv_configuration; fi
-    run_e2e_test
+    if [[ ! "${PRIVATE_TESTING}" == "true" ]]; then
+        chmod +x "${SCRIPT_ROOT}/run-k8s-e2e-test.sh"
```

Comment:
This line shouldn't be needed. Git and preserve file permissions.

Reply:
I ran into permission issue locally without this. I will double check.